### PR TITLE
added scheduler to reduce learning rate on plateau

### PR DESCRIPTION
**Pull request by zbuckley**
Adds a scheduler which reduces that learning rate when training attempt seems to have optimized as best it can at the current rate.

### CODE REVIEW

File: Code/training_team8.py [hunk 1]
```diff
@@ -59,6 +55,8 @@
 # TODO: Need to update this? possibly not.
 criterion = nn.BCEWithLogitsLoss()
 
+scheduler = lr_scheduler.ReduceLROnPlateau(optimizer, 'min')
```
Comment (zbuckley) [inline Code/training_team8.py]: Make this verbose

**Review by josh-bernstein618 — APPROVED** [review]
